addons/website_portal/views/templates.xml

```diff
@@ -10,9 +10,32 @@
 
     <template id="account" name="Account">
         <t t-call="website.layout">
+            <div class="container mt16">
+                <div class="navbar navbar-default">
+                    <div class="navbar-header">
+                        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#oe-help-navbar-collapse">
+                            <span class="sr-only">Toggle navigation</span>
+                            <span class="icon-bar"></span>
+                            <span class="icon-bar"></span>
+                        </button>
+                        <a class="navbar-brand" href="/my/home">
+                            My Account
+                        </a>
+                    </div>
+                    <div class="collapse navbar-collapse" id="oe-help-navbar-collapse">
+                        <ul class="nav navbar-nav o-portal-submenu">
```

Comment:
In Odoo, we use 'underscore' notation for css class. I don't think this has changed.
Moreover, please, prefix all css class by `o_my_` for this module.